README.md

```diff
@@ -283,10 +283,34 @@ for managing extensions and reporting VM status. The core functionality of the a
 is contained in the Extension Handler, and we encourage users to enable this option 
 in order to maintain an up to date version.
 
+_Note_: 
+1. Setting this option to 'n' will not only disable updates to the latest version but also revert the current version of the Extension Handler back to the version that was shipped with the image.
+2. We introduced a new option, AutoUpdate.UpdateToLatestVersion, to control agent updates. This option is available starting from version 2.10.0.8 and higher. We strongly encourage users to utilize the new option instead of AutoUpdate.Enabled.
+3. The AutoUpdate.Enabled flag is retained for legacy purposes. If an image comes with a pre-installed agent version below 2.10.0.8, this flag will be used for the first agent update. Once the agent is updated to version 2.10.0.8 or higher, the new flag AutoUpdate.UpdateToLatestVersion takes effect
+
 On most distros the default value is 'y'.
 
 For more information on the agent version, see our [FAQ](https://github.com/Azure/WALinuxAgent/wiki/FAQ#what-does-goal-state-agent-mean-in-waagent---version-output).
 
+#### __AutoUpdate.UpdateToLatestVersion__
+
+_Type: Boolean_
+_Default: y_
+
+Enables automatic updates of the Extension Handler to the latest version published by Microsoft.
+If this option is set to 'n', the Extension Handler will remain pinned to the latest installed version that was on the VM, and future updates will not be received.
+We recommend users enable this option to ensure they have the latest version.
+
+_Note_:
+1. This option becomes effective from version 2.10.0.8 onwards.
+2. Applies only when AutoUpdate.Enabled is set to 'y'.
```

Comment:
Should we mention here it only applies when AutoUpdate.Enabled is 'y' OR not in the conf at all?

Reply:
Or alternatively, it is only applied if AutoUpdate.Enabled is not 'n'